about.html

```diff
@@ -0,0 +1,183 @@
+<!-- @format -->
+
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Bin Tech</title>
+		<link rel="stylesheet" href="styles.css" />
+	</head>
+	<body>
+		<header class="header">
+			<section class="top-nav desktop">
+				<ul class="social-icon container">
+					<li>
+						<a href="https://linkedin.com/in/binhussen">
+							<img src="./assets/icon/icon-linkedin.svg" alt="Linkedin" />
+						</a>
+					</li>
+					<li>
+						<a href="https://github.com/binhussen">
+							<img src="./assets/icon/icon-github.svg" alt="Github" />
+						</a>
+					</li>
+
+					<li>
+						<a href="https://twitter.com/binhussens">
+							<img src="./assets/icon/icon-twitter.svg" alt="Twitter" />
+						</a>
+					</li>
+					<li>English</li>
+				</ul>
+			</section>
+			<section class="nav container">
+				<a href="index.html">
+					<img
+						src="./assets/icon/icon-logo.png"
+						alt="Bin Tech"
+						class="logo desktop"
+				/></a>
+				<nav>
+					<ul class="nav-menu desktop">
+						<li class="nav-item"><a href="index.html">Home</a></li>
+						<li class="nav-item select"><a href="about.html">About</a></li>
+						<li class="nav-item"><a href="#">Program</a></li>
+						<li class="nav-item"><a href="#">Join Us</a></li>
+						<li class="nav-item">
+							<a href="#">Contact Us</a>
+						</li>
+					</ul>
+					<button type="button" class="hamburger mobile">
+						<span class="bar"></span>
+						<span class="bar"></span>
+						<span class="bar"></span>
+					</button>
+				</nav>
+			</section>
```

Comment:
- [ ] I would like to suggest you use the `<div>` tag instead of the `<section>` tag for your menu because HTML `<section>` Tag. Section tag defines the section of documents such as chapters, headers, footers or any other sections :+1: 

Example of implementation

```
   <div class="topnav">
        /// Social media link 
    </div>
    <div class="bottomnav">
        <nav>
           ----------
       </nav>
    </div>

```

Reply:
Thanks, @codecaiine for your feedback, I will modify it.